python/src/aiconfig/editor/client/src/components/prompt/prompt_outputs/PromptOutputWrapper.tsx

```diff
@@ -0,0 +1,59 @@
+import { ActionIcon, CopyButton, Flex, Tooltip } from "@mantine/core";
+import {
+  IconBraces,
+  IconBracesOff,
+  IconCheck,
+  IconCopy,
+} from "@tabler/icons-react";
+import { Output } from "aiconfig";
+import { memo, useState } from "react";
+import JSONOutput from "./JSONOutput";
+
+type Props = {
+  children: React.ReactNode;
```

Comment:
If we want to support rendering custom outputs as children here we can update to a render prop here later (e.g. renderOutput: (output: Output) => React.Element)